src/System Application/App/Extension Management/src/ExtensionDeploymentStatus.Page.al

```diff
@@ -99,6 +100,49 @@ page 2508 "Extension Deployment Status"
                     ExtnDeploymentStatusDetail.Run();
                 end;
             }
+            action(Refresh)
+            {
+                ApplicationArea = All;
+                ToolTip = 'Refresh the deployment details.';
+                Image = Refresh;
+
+                trigger OnAction()
+                var
+                    NavAppTenantOperationTable: Record "NAV App Tenant Operation";
+                    ExtensionOperationImpl: Codeunit "Extension Operation Impl";
+                begin
+                    NavAppTenantOperationTable.CopyFilters(Rec);
```

Comment:
I think this doesn't copy marked records (rows you select with cntrl)

This one should (although it also copies the value of the record)
https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/methods-auto/record/record-copy-method

I

Reply:
Without marked records, all the rows in view will be refreshed even if you select some

Reply:
I might be wrong though and copy filters does copy the marked records, it should be straightforward to test by selecting rows and calling the action